core-bundle/src/Resources/contao/library/Contao/TemplateInheritance.php

```diff
@@ -141,6 +141,9 @@ public function inherit()
 			$blnDebug = System::getContainer()->getParameter('kernel.debug');
 		}
 
+		// Replace insert tags
+		$strBuffer = System::getContainer()->get('contao.insert_tag.parser')->replace($strBuffer);
```

Comment:
As discussed in the Contao call, we want to wrap this with `instanceof FrontendTemplate` so it is only executed for front end templates.

Reply:
Doen in d8a817b80c1813e9043358aee83cf25c33dd4d38